cmd/soroban-cli/src/commands/plugin/search.rs

```diff
@@ -0,0 +1,82 @@
+use serde::{Deserialize, Serialize};
+use std::io::Write;
+use termcolor::{Color, ColorChoice, ColorSpec, StandardStream, WriteColor};
+
+use crate::{commands::global, print::Print, utils::http};
+
+use super::super::config::locator;
+
+#[derive(thiserror::Error, Debug)]
+pub enum Error {
+    #[error(transparent)]
+    Config(#[from] locator::Error),
+
+    #[error("unable to retrieve the list of plugins from GitHub")]
+    Http(#[from] reqwest::Error),
+
+    #[error(transparent)]
+    Io(#[from] std::io::Error),
+}
+
+#[derive(Debug, clap::Parser, Clone)]
+#[group(skip)]
+pub struct Cmd;
+
+impl Cmd {
+    pub async fn run(&self, global_args: &global::Args) -> Result<(), Error> {
+        let print = Print::new(global_args.quiet);
+        let url =
+            "https://api.github.com/search/repositories?q=topic:stellar-cli-plugin&per_page=100";
```

Comment:
I think we should alter the search query:
- sort by stars with `&sort=stars&order=desc`
- exclude forks with `fork:false`
- exclude archived repos with `archived:false`

Reply:
Great! Didn't think about that!